src/scripts.js

```diff
@@ -194,8 +209,22 @@ function displayRecipes(recipes) {
      `;
   });
 
-  createCardEventListeners();
-  createButtonEventListeners();
+  recipeCards = document.querySelectorAll('.recipe-card-js');
```

Comment:
We put these back in and are doing away with the helper functions. Since we have to reassign variables in the helper functions back in this function, we're thinking that this is actually a better choice.